elytron/src/test/java/org/wildfly/extension/elytron/SubsystemTransformerTestCase.java

```diff
@@ -121,6 +121,12 @@ public void testRejectingTransformersEAP720() throws Exception {
                 .addFailedAttribute(SUBSYSTEM_ADDRESS.append(PathElement.pathElement(ElytronDescriptionConstants.SIZE_ROTATING_FILE_AUDIT_LOG, "audit6")),
                         new FailedOperationTransformationConfig.NewAttributesConfig(AuditResourceDefinitions.AUTOFLUSH)
                 )
+                .addFailedAttribute(SUBSYSTEM_ADDRESS.append(PathElement.pathElement(ElytronDescriptionConstants.SYSLOG_AUDIT_LOG, "audit7")),
+                        new FailedOperationTransformationConfig.NewAttributesConfig(AuditResourceDefinitions.RECONNECT_ATTEMPTS)
+                )
+                .addFailedAttribute(SUBSYSTEM_ADDRESS.append(PathElement.pathElement(ElytronDescriptionConstants.SYSLOG_AUDIT_LOG, "audit8")),
+                        new FailedOperationTransformationConfig.NewAttributesConfig(AuditResourceDefinitions.SYSLOG_FORMAT)
+                )
```

Comment:
XML elements corresponding to "audit7" and "audit8" should also be added in elytron-transformers-4.0-reject.xml